businesscentral/image.yaml

```diff
@@ -0,0 +1,26 @@
+name: "jboss-bpmsuite-7/bpmsuite70-businesscentral"
+version: "7.0.0"
+from: "jboss-eap-7/eap70:latest"
+user: 185
+envs:
+    information:
+        - name: "JBOSS_PRODUCT"
+          value: "bpmsuite"
+        - name: "JBOSS_BPMSUITE_VERSION"
+          value: "7.0.0.CAP01"
+        - name: "PRODUCT_VERSION"
+          value: "7.0.0.CAP01"
+cmd:
+    - "/opt/eap/bin/standalone.sh"
+    - "-b"
+    - "0.0.0.0"
+    - "-c"
+    - "standalone-full.xml"
+scripts:
+    - package: businesscentral
+      exec: install
+sources:
+    - url: jboss-bpmsuite-7.0.0.CAP01-business-central-eap7.zip
```

Comment:
No, it does not need to match. We use the hash only to retrieve the correct file.